### ISSUE TITLE
Fun Commands

### ISSUE THREAD

**Issue by ArcaneDisgea** [poieten-oikos/venat#7]
These commands are kinda meant to be lowhanging fruit just haha funny type commands. No real priority, super basic and relatively easy to implement ideally. This list is not exhaustive by any means and will kind of just be bait so we can say we did something on the days we don't feel like doing anything.

- [ ] Music
- [ ] Dice Rolls
- [ ] Random cat/dog/animal
- [ ] Random meme?
- [ ] Random gif optional arg for a theme? `/gif water` discord kinda has this built in tho so not really necessary unless we want to add other providers outside of the two discord offer, tenor/giphy.

**Comment by erroch** [comment]
A couple of items that have been on my todo list that I've been too busy to ever get to

- [ ] FFXIV NPC Name Generator  (By subrace / gender)
- [ ] Tournament bracket generator / tracker (Might be too big of an ask for 'fun stuff')
- [ ] Quote system (usually for out of context)

**Comment by philpax** [comment]
Not sure if "fun" specifically, but getting time/weather for a RL location?

**Comment by ArcaneDisgea** [comment]
Yeah I think another category/module called informational should cover those. Would contain commands like dictionary look ups, weather, temperatures, time in locations, reminders, polls, user info, etc.
